EADModule6/src/main/java/com/metacube/REST/dao/ReadPropertiesFile.java

```diff
@@ -0,0 +1,27 @@
+package com.metacube.REST.dao;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * This class is used for read properties file.
+ */
+
+public class ReadPropertiesFile {
+
+	public Properties readFile() throws IOException {
```

Comment:
* This method should be static.
* This should receive file name as parameter.
* And this Class should be in Util folder